Scripts/sdk_utilities.rb

```diff
@@ -0,0 +1,95 @@
+require 'erb'
+require 'open3'
+
+class TemplateBuilder
+  def initialize(version, commit_sha, license, homepage_url, authors, ios_deployment_target)
+    @version = version
+    @commit_sha = commit_sha
+    @license = license
+    @homepage_url = homepage_url
+    @authors = authors
+    @ios_deployment_target = ios_deployment_target
+  end
+
+  def build_and_write(template_path, file_name, output_dir)
+    template = File.read(template_path)
+    renderer = ERB.new(template)
+    result = renderer.result(get_binding)
+    File.open("#{output_dir}#{file_name}", 'w+') do |file|
+      file.write(result)
+    end
+  end
+
+  private
+
+  def get_binding
```

Comment:
do you need this? I think `binding` should be available in this context already, the common case for a getter for binding is to make it public (which I don't think you'd need?)

Reply:
Removed! 🙏